state.go

```diff
@@ -164,7 +164,8 @@ type StateData struct {
 
 const (
 	// name of key that state data is stored under across reboots
-	stateDataKey = "state"
+	stateDataKey       = "state"
+	stateDataKeyCustom = "state-custom"
```

Comment:
Suggest renaming to "uncommitted" instead of "custom" to make it a bit more clear.

Reply:
Fixing!